config/locales/en.yml

```diff
@@ -2,6 +2,19 @@ en:
   time:
     formats:
       admin: "%R on %d/%m/%Y"
+  courses:
+    alternative_names:
```

Comment:
Would it be possible to change the key to identifier? To match this https://github.com/DFE-Digital/npq-registration/pull/766/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R236? 

It'd work into the consistent naming system that Emma and I are introducing through the app.

Reply:
I actually wondered whether this is something that should be part of the database model? It feels a bit odd to have a part of naming in the database and the rest in translations...

Reply:
This is actually something we're specifically moving away from, the names being in the DB was causing several issues. Moving to just holding the identifier in the DB and then names in the locale file has made things easier. I've taken what you've said on board in that PR and swapped course.identifier with course.name.

Reply:
The issues were down to the DB values causing inconsistencies across this and the ECF app and needing to refresh the DB to make changes. Having the identifier in the DB and the locales handling how that is converted into what the user sees makes deploying and managing things easier.